addons/content-index/src/main/java/org/commonjava/indy/content/index/DefaultContentIndexManager.java

```diff
@@ -72,6 +72,7 @@
     @Inject
     private BasicCacheHandle<IndexedStorePath, IndexedStorePath> contentIndex;
 
+    //FIXME: Seems no cache register this listener?
```

Comment:
Ouch, that seems like it needs to be fixed. Won't that prevent NFC entries from expiring properly?

Reply:
I think that the related nfc cleaning function is not working for all ispn cache related removing now as this listener is not registered.